client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-sections.tsx

```diff
@@ -0,0 +1,73 @@
+import { NavigatorHeader } from '@automattic/onboarding';
+import {
+	Button,
+	__experimentalVStack as VStack,
+	__experimentalUseNavigator as useNavigator,
+} from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { NAVIGATOR_PATHS } from './constants';
+import { PATTERN_ASSEMBLER_EVENTS } from './events';
+import { useScreen } from './hooks';
+import NavigatorTitle from './navigator-title';
+import PatternCategoryList from './pattern-category-list';
+import { Pattern, Category } from './types';
+
+interface Props {
+	categories: Category[];
+	patternsMapByCategory: { [ key: string ]: Pattern[] };
+	onContinueClick: () => void;
+	recordTracksEvent: ( name: string, eventProperties?: any ) => void;
+}
+
+const ScreenSections = ( {
+	categories,
+	patternsMapByCategory,
+	onContinueClick,
+	recordTracksEvent,
+}: Props ) => {
+	const translate = useTranslate();
+	const { title } = useScreen( 'sections' );
+	const { params, goTo } = useNavigator();
+	const selectedCategory = params.categorySlug as string;
+
+	const onSelectSectionCategory = ( category: string ) => {
+		const nextPath =
+			category !== selectedCategory
+				? `${ NAVIGATOR_PATHS.SECTIONS }/${ category }`
+				: NAVIGATOR_PATHS.SECTIONS;
+
+		goTo( nextPath, { replace: true } );
+		recordTracksEvent( PATTERN_ASSEMBLER_EVENTS.CATEGORY_LIST_CATEGORY_CLICK, {
+			pattern_category: category,
+		} );
+	};
+
+	return (
+		<>
+			<NavigatorHeader
+				title={ <NavigatorTitle title={ title } /> }
+				description={ translate(
+					'Find the right patterns for you be exploring the list of categories below'
```

Comment:
Is this string already translated? If not WDYT about using the previous one `Find the section patterns for your homepage by exploring the categories below.` and we can update the copy together with https://github.com/Automattic/wp-calypso/issues/81307?

Reply:
I thought it was and I'm wrong 🥲 Use `Find the section patterns for your homepage by exploring the categories below.` for non-English locale.